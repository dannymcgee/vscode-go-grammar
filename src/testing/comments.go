// This is a line comment
/*
This is a block comment
*/
