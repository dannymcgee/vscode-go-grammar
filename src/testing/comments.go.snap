// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`comments should work 1`] = `

-> // This is a line comment
 | ^^ punctuation.definition.comment.line.go  comment.line.go
 |   ^^^^^^^^^^^^^^^^^^^^^^^ comment.line.go
-> /*
 | ^^ punctuation.definition.comment.block.begin.go  comment.block.go
-> This is a block comment
 | ^^^^^^^^^^^^^^^^^^^^^^^^ comment.block.go
-> */
 | ^^ punctuation.definition.comment.block.end.go  comment.block.go
-> 
`;
